Content.Shared/Containers/ItemSlot/ItemSlotsSystem.cs

```diff
@@ -205,9 +206,31 @@ private void OnInteractUsing(EntityUid uid, ItemSlotsComponent itemSlots, Intera
                 if (!CanInsert(uid, args.Used, args.User, slot, swap: slot.Swap, popup: args.User))
                     continue;
 
-                // Drop the held item onto the floor. Return if the user cannot drop.
-                if (!_handsSystem.TryDrop(args.User, args.Used, handsComp: hands))
-                    return;
+                slots.Add(slot);
+            }
+
+            if (slots.Count == 0)
+                return;
+
+            slots.Sort(static (a, b) =>
+            {
+                var aEnt = a.ContainerSlot?.ContainedEntity;
+                var bEnt = b.ContainerSlot?.ContainedEntity;
+                if (aEnt == null && bEnt == null)
+                    return a.Priority.CompareTo(b.Priority);
+
+                if (aEnt == null)
+                    return -1;
+
+                return 1;
+            });
+
```

Comment:
Should this sort after the drop.

Reply:
Fixed in [`0b1165e` (#28203)](https://github.com/space-wizards/space-station-14/pull/28203/commits/0b1165e469767b14f705aa69f13e483425c456c8)